browser/ui/webui/brave_welcome_ui.cc

```diff
@@ -171,6 +180,25 @@ void WelcomeDOMHandler::HandleImportNowRequested(
                                              chrome::kImportDataSubPage);
 }
 
+void WelcomeDOMHandler::GetDefaultBrowser(const base::Value::List& args) {
+  CHECK_EQ(1U, args.size());
+  const auto& callback_id = args[0].GetString();
+  AllowJavascript();
+
+  base::MakeRefCounted<shell_integration::DefaultProtocolClientWorker>(
```

Comment:
How about waiting cr111 merge and rename this to `DefaultSchemeClientWorker` as its name is chaged in cr111.
It seems cr111 is about to merge very soon.

Reply:
ok